src/Contract/Transaction.purs

```diff
@@ -253,31 +314,69 @@ derive newtype instance Eq BalancedSignedTransaction
 instance Show BalancedSignedTransaction where
   show = genericShow
 
+-- |Like 'balanceAndSignTx, but for more than one transaction.
+-- This function may throw errors through the contract Monad.
+-- If successful, transaction inputs will be locked afterwards.
+-- If you want to re-use them in the same 'QueryM' context, call
+-- 'unlockTransactionInputs'.
+balanceAndSignTxs
+  :: forall (r :: Row Type)
+   . NonEmptyArray UnattachedUnbalancedTx
+  -> Contract r (NonEmptyArray BalancedSignedTransaction)
+balanceAndSignTxs txs = do
```

Comment:
Can we please make this function covered by a new example in `examples/` dir?

Reply:
Yes, `SignMultiple`. Based on `Pkh2Pkh`